website/src/pages/Blog/styles.ts

```diff
@@ -3,12 +3,20 @@ import { makeStyles, Theme } from "@material-ui/core/styles";
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
     position: "relative",
-    backgroundImage: `url(${"/assets/images/faq_background.png"})`,
+    backgroundImage: "url(/Images/png/faq_background.png)",
     textAlign: "center",
     backgroundRepeat: "no-repeat",
     backgroundPosition: "center top",
     backgroundSize: "cover",
     padding: theme.spacing(16, 5, 0),
+    [theme.breakpoints.down("xs")]: {
+      padding: theme.spacing(5, 5),
+      marginTop: theme.spacing(10),
+      backgroundImage: "url(/Images/png/faq_background_mobile.png)",
+      textAlign: "center",
+      backgroundRepeat: "no-repeat",
+      backgroundSize: "cover",
```

Comment:
same styles dont have to repeat

Reply:
Removed `text-align`. Kept the rest of the styling. When the background image is changed using breakpoint, we need to provide other background image styling as well as that will not be inherited